features/zoeken.feature

```diff
@@ -7,115 +7,107 @@ Functionaliteit: zoeken van WOZ-objecten
+  #     Als ik een WOZ-object zoek met /wozobjecten?postcode=8000GB&huisnummer=1&huisletter=b
+  #     Dan bevat het antwoord niet het WOZ-object met identificatie "000500000001"
+
+  #   Voorbeeld: zoek WOZ-object met postcode en huisnummer en onjuiste huisnummertoevoeging
+  #     Gegeven de LV WOZ kent een WOZ-object met objectnummer "000500000001"
+  #     En de aanduiding WOZ-object heeft postcode "8000GB" en huisnummer 1 en huisletter "a" en huisnummertoevoeging "bis"
+  #     Als ik een WOZ-object zoek met /wozobjecten?postcode=8000GB&huisnummer=1&huisnummertoevoeging=andere
+  #     Dan bevat het antwoord niet het WOZ-object met identificatie "000500000001"
 
     Voorbeeld: zoek WOZ-object met nummeraanduidingIdentificatie van de aanduiding
-      Gegeven de LV WOZ kent een WOZ-object met objectnummer "000500000001"
-      En de aanduiding WOZ-object heeft nummeraanduidingIdentificatie "0005200000009179"
-      Als ik een WOZ-object zoek met /wozobjecten?nummeraanduidingIdentificatie=0005200000009179
-      Dan bevat het antwoord het WOZ-object met identificatie "000500000001"
+      Gegeven de LV WOZ kent een WOZ-object met objectnummer "800000051111"
+      En de aanduiding WOZ-object heeft nummeraanduidingIdentificatie "8513200000050111"
+      Als ik een WOZ-object zoek met "/wozobjecten?nummeraanduidingIdentificatie=8513200000050111"
+      Dan bevat het antwoord het WOZ-object met identificatie "800000051111"
 
-  Regel: zoeken met huisletter en huisnummertoevoeging is case-insensitive
+  # Rule: zoeken met huisletter en huisnummertoevoeging is case-insensitive
 
-    Voorbeeld: zoek WOZ-object met postcode en huisnummer en huisletter en huisnummertoevoeging in andere case dan in de aanduiding
-      Gegeven de LV WOZ kent een WOZ-object met objectnummer "000500000001"
-      En de aanduiding WOZ-object heeft postcode "8000GB" en huisnummer 1 en huisletter "a" en huisnummertoevoeging "bis"
-      Als ik een WOZ-object zoek met /wozobjecten?postcode=8000GB&huisnummer=1&huisletter=A&huisnummertoevoeging=BIS
-      Dan bevat het antwoord het WOZ-object met identificatie "000500000001"
+  #   Voorbeeld: zoek WOZ-object met postcode en huisnummer en huisletter en huisnummertoevoeging in andere case dan in de aanduiding
+  #     Gegeven de LV WOZ kent een WOZ-object met objectnummer "000500000001"
+  #     En de aanduiding WOZ-object heeft postcode "8000GB" en huisnummer 1 en huisletter "a" en huisnummertoevoeging "bis"
+  #     Als ik een WOZ-object zoek met /wozobjecten?postcode=8000GB&huisnummer=1&huisletter=A&huisnummertoevoeging=BIS
+  #     Dan bevat het antwoord het WOZ-object met identificatie "000500000001"
 
-  Regel: Zoeken van WOZ-objecten levert lege property _embedded wanneer er niks gevonden wordt
+  Rule: Zoeken van WOZ-objecten levert lege property _embedded wanneer er niks gevonden wordt
 
     Voorbeeld: zoeken met adresseerbaar object identificatie vindt geen WOZ-object
```

Comment:
Idem.

Reply:
Deze heb ik vergeten te verwijderen. Hij staat al  in onderstaande Abstract Scenario